### ISSUE TITLE
Allow creator of a community to set the name for their currency

### ISSUE THREAD

**Issue by aquabu** [citizencode/swarmbot#58]
...and provide a description.

**Comment by fractastical** [comment]
I actually implemented this awhile ago, but no description, it's in the admin controller

**Comment by aquabu** [comment]
Should it be listed as an advanced command?

**Comment by fractastical** [comment]
Yep, ideally we differentiate between commands that are for an owner of a
community only and advanced commands that are for any user
